plugin/tests/test-extractor.php

```
@@ -0,0 +1,369 @@
+        // Test: ID Collision Prevention
+        echo "\nTest: ID Collision Prevention\n";
+        echo "------------------------------\n";
+        $this->test(
+            "Different IDs for same key in different subfolders (standalone-apps)",
+            strpos($output, 'id="unraid-standalone-apps-app-entry"') !== false
+        );
+        $this->test(
+            "Different IDs for same key in different subfolders (other)",
+            strpos($output, 'id="unraid-other-app-entry"') !== false
+        );
+        $appEntryCount = substr_count($output, 'id="unraid-') - substr_count($output, 'id="unraid-ui-');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$appEntryCount'. (Unused Code Rules)

(UnusedLocalVariable)